mirrulations-dashboard/src/mirrdash/static/index.js

```diff
@@ -86,7 +86,8 @@ const updateDashboardData = () => {
             mongo,
             redis,
             work_generator,
-            work_server
+            work_server,
+            num_extractions_done
```

Comment:
For future reference this is supposed to be a container where we look at the status of the docker container